indico/web/templates/forms/_access_list.html

```diff
@@ -0,0 +1,25 @@
+<div class="i-box-group vert">
+    {% if users or groups %}
+        <div class="i-box just-group-list">
+            <div class="i-box-content">
+                <ul class="group-list">
+                    {% for user in users %}
+                        <li class="icon-user">{{ user.full_name }}</li>
+                    {% endfor %}
+                    {% for group in groups %}
+                        <li class="icon-users">{{ group.name }}</li>
+                    {% endfor %}
+                </ul>
+            </div>
+        </div>
+    {% endif %}
+    <div class="action-box message-only for-form highlight" style="display: block;">
+        <div class="section">
+            <div class="text">
+                <div>
+                    {% trans %}Users with modification access over this resource and any parent resources will also have access{% endtrans %}.
```

Comment:
What about making this dialog a bit more fancy? Instead of hiding people who are directly on the ACL you could show them separately; same for managers. The uber-fancy option would be showing the protection inheritance graph and which people have access granted from that place. But I guess that'd be overkill for now.

But maybe it'd be better to keep it as simple as it is atm for now and consider improving it after 2.0...